packages/api/src/user/types.ts

```diff
@@ -7,6 +7,10 @@ export interface HasBalanceArgs {
   amount: string
 }
 
+export interface TokenSymbolType {
```

Comment:
As you can see, we're following some naming conventions in the API types. You have to do the following:

```suggestion
/*
Example balance response is:

  token: {
    uuid: 'fa209a15-c742-5ecd-9793-745016899ad6',
    name: 'Decimals16',
    symbol: 'QDEC16',
    decimals: 16,
    logo: 'https://roll-external-link-icon.s3.amazonaws.com/roll-test-icon/roll_icon.png',
    contractAddress: '0xd5e7214AcF2447c40AC866e9C5195fc0e419bd0B',
    currentSupply: '1999999.9999999983515342',
    totalSupply: '10000000'
  },
  amount: '0.6',
  updatedAt: '2023-06-29T10:00:56.329565Z'
*/

export interface PlatformUserTokenBalanceResponseData {
  // Given the example response above, describe this type
}
```

Reply:
Thank you! Will do.